src/app.jsx

```diff
@@ -153,12 +64,22 @@ class App extends React.Component {
 }
 
 $.get('config.json', (config) => {
+  let Router = ReactRouter.Router,
+    Route = ReactRouter.Route, browserHistory = ReactRouter.browserHistory
+
   var query = parseQuery(window.location.search);
   if(config.piwik && ! query['code']) {
     trackPiwik(config.piwik);
   }
-  window.app = React.render(
-    <App config={config} query={query} />,
+  window.__app_config = config;
```

Comment:
We should probably refactor the configuration, pass it via `props` all the time. But after we get this pull request merged.